### PR TITLE
migrates to androidx

### PR DESCRIPTION
**Pull request by fahadwajed**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -27,8 +27,8 @@ dependencies {
   implementation 'androidx.appcompat:appcompat:1.0.2'
   implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
   implementation "androidx.cardview:cardview:1.0.0"
-  implementation 'com.android.support:appcompat-v7:28.0.0'
-  implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+  implementation 'androidx.appcompat:appcompat:1.0.0'
+  implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```
Comment (bfranks) [inline app/build.gradle]: AndroidX libraries are directly above this I think we can remove this duplication.

**Comment by bfranks** [comment]
PR into master we need to make sure the version bump happens. to not overwrite the version.

File: app/build.gradle (hunk 1 above)
Comment (fahadwajed) [reply app/build.gradle]: removed

**Comment by fahadwajed** [comment]
> PR into master we need to make sure the version bump happens. to not overwrite the version.

it's checked against master.

**Pull request closed without merging**